tests/compiletest.rs

```diff
@@ -66,8 +67,6 @@ fn test_config(target: &str, path: &str, mode: Mode, with_dependencies: bool) ->
         }
     }
     program.args.push("-Zui-testing".into());
-    program.args.push("--target".into());
-    program.args.push(target.into());
```

Comment:
From the CI errors, removing `--target` doesn't seem to be right.

Reply:
yea ^^ looks like it